lib/dogapi/common.rb

```diff
@@ -141,6 +142,12 @@ def prepare_request(method, url, params, body, send_json, with_app_key)
       unless should_set_api_and_app_keys_in_params?(url)
         req['DD-API-KEY'] = @api_key
         req['DD-APPLICATION-KEY'] = @application_key if with_app_key
+        req['User-Agent'] = 'dogapi-rb/%{version} (ruby %{ruver}; os %{os}; arch %{arch})' % {
+          :version => VERSION,
+          :ruver => RUBY_VERSION,
+          :os => RbConfig::CONFIG['host_os'].downcase,
+          :arch => RbConfig::CONFIG['host_cpu']
+        }
       end
```

Comment:
```suggestion
      end
      req['User-Agent'] = 'dogapi-rb/%{version} (ruby %{ruver}; os %{os}; arch %{arch})' % {
        :version => VERSION,
        :ruver => RUBY_VERSION,
        :os => RbConfig::CONFIG['host_os'].downcase,
        :arch => RbConfig::CONFIG['host_cpu']
      }
```

Reply:
It feels like the User-Agent value could be a constant.

Reply:
True, I can make it a constant.